ui/page/main_page.go

```diff
@@ -505,9 +535,18 @@ func (mp *MainPage) LayoutTopBar(gtx layout.Context) layout.Dimensions {
 											})
 									}),
 									layout.Rigid(func(gtx C) D {
-										return layout.Center.Layout(gtx, func(gtx C) D {
-											return components.LayoutBalance(gtx, mp.Load, mp.totalBalance.String())
+										return mp.hideMyBalance(gtx)
+									}),
+									layout.Rigid(func(gtx C) D {
+										icon := decredmaterial.MustIcon(widget.NewIcon(icons.ActionVisibility))
+										if mp.hiddenValue {
+											icon = decredmaterial.MustIcon(widget.NewIcon(icons.ActionVisibilityOff))
+										}
+										mp.hideBalance.Icon = icon
```

Comment:
```suggestion
										mp.hideBalance.Icon = mp.Icons.ConcealIcon
										if mp.isBalanceHidden {
											mp.hideBalance.Icon = mp.Icons.RevealIcon
										}	
```

you already declared the needed icons on the load.go page, it's redundant to re-initialise the icons here again